modules/keyword/taunt/taunt.gd

```diff
@@ -5,7 +5,7 @@ extends Node
 func _ready() -> void:
 	Modules.register_hooks(handler)
 	
-	Modules.register_keyword(&"Taunt")
+	KeywordModule.register_keyword(&"Taunt")
```

Comment:
Using modules directly is **_VERY BAD_**. The only exception is when using a base module inside of a child module. E.g. Using `KeywordModule` in the `Taunt` keyword, like we are doing here.